.github/workflows/release.yml

```diff
@@ -22,21 +21,25 @@ jobs:
         with:
           node-version-file: '.nvmrc'
           cache: npm
-      - name: install
-        run: npm ci || npm install
-      - run: npm test
-      - uses: fregante/daily-version-action@v1
+      - run: npm ci
+      - name: Test and build
+        run: npm test
+      - uses: fregante/daily-version-action@v2
         name: Create tag if necessary
-        id: daily-version
-      - if: steps.daily-version.outputs.created
+      - name: Update extension’s meta
+        run: npx dot-json@1 $DIRECTORY/manifest.json version $DAILY_VERSION
```

Comment:
`$DAILY_VERSION` is also missing here, so the manifest.json still contains 0.0.0